Tasks/Common/SonarQubeHelpers/SonarQubeHelper.ps1

```diff
@@ -12,22 +12,30 @@ function EscapeArg
 
 
 # Set a variable in a property bag that is accessible by all steps
-# To retrieve the variable use $val = Get-Variable $distributedTaskContext "varName"
+# To retrieve the variable use GetTaskContextVariable
+# The variable will be available in both the current and the subsequent tasks
 function SetTaskContextVariable
```

Comment:
The variable is available in the current task via env variables and in subsequent tasks via Set-VstsTaskVariable. There is a breaking change that in the new coreclr agent - setting a var directly using Set-VstsTaskVariable (which is a task lib API call), no longer makes it available in the current step.